Document homeassistant hardware (#24967)
home-assistant/home-assistant.github.io: source/_integrations/homeassistant_hardware.markdown

---
title: Home Assistant Hardware

> description: Home Assistant Hardware provides functionality shared between Home Assistant Yellow and Home Assistant Sky Connect.

Reviewer: ```suggestion
description: Home Assistant Hardware provides functionality shared between Home Assistant Yellow and Home Assistant SkyConnect.
```

> ha_release: 2022.8

Reviewer: ```suggestion
ha_release: 2022.12
```

ha_category:
  - Other
ha_codeowners:
  - '@home-assistant/core'
ha_domain: homeassistant_hardware
ha_integration_type: system
ha_quality_scale: internal
---

> The Home Assistant Home Assistant Hardware integration provides functionality
> shared between Home Assistant Yellow and Home Assistant Sky Connect.

Reviewer: ```suggestion
The Home Assistant Hardware integration provides functionality
shared between Home Assistant Yellow and Home Assistant SkyConnect.
```
